bokehjs/src/lib/models/tools/edit/box_edit_tool.ts

```diff
@@ -59,73 +59,73 @@ export class BoxEditToolView extends EditToolView {
       return
     const {glyph} = renderer
     const cds = renderer.data_source
-    const [x0, x1] = renderer_view.coordinates.x_scale.r_invert(sx0, sx1)
-    const [y0, y1] = renderer_view.coordinates.y_scale.r_invert(sy0, sy1)
+    const [rx0, rx1] = renderer_view.coordinates.x_scale.r_invert(sx0, sx1)
+    const [ry0, ry1] = renderer_view.coordinates.y_scale.r_invert(sy0, sy1)
     const fields = (() => {
```

Comment:
This would have caught the problem at the type-level:
```suggestion
    const fields: {[key: string]: number} | null = (() => {
```